### ISSUE TITLE
Errors were found in paper data reproduction

### ISSUE THREAD

**Issue by Zehao-Lee** [VisualComputingInstitute/2D_lidar_person_detection#18]
Dear author, when I experimented with the DROW3 model training data set according to the tips in the paper, I found that the error detection rate of this method was too high, and the target was detected where there was no point cloud data.
The whole experiment process is completed under laser_link coordinate system, may I ask whether it is the problem that the coordinate system does not change or other reasons?
![74ffe65a3ba3e49cdfe0ff61bfd603a](https://github.com/VisualComputingInstitute/2D_lidar_person_detection/assets/143074044/bafa408f-8d48-4783-a32b-74b2d30a95e9)

**Comment by Pandoro** [comment]
I'm not a 100% sure what you mean with the DROW3 model training data set. Did you play around with the training yourself, or just with an existing model?

I can't fully remember what frame we used. In any case the predictions are always made with reference to the scan, whereas laser_link might be moving in the world according to robot odometry?

Apart from that, there could me multiple reasons why you are seeing the predictions you are seeing. In general I agree that it doesn't look correct, but given that DROW3 does use some temporal information, it might still predict a person, even if it isn't visible in the exact frame. In general we trained and evaluated the models without any ROS code and simply used a dumped version of the data. But I think other people have used the ROS node successfully, so in principle it should be working. Are you seeing similar effects with DR-SPAAM?
